tests/e2e/tests/regression/workbench/command-results.e2e.ts

```diff
@@ -0,0 +1,74 @@
+import { addNewStandaloneDatabase } from '../../../helpers/database';
+import { WorkbenchPage } from '../../../pageObjects/workbench-page';
+import { MyRedisDatabasePage, UserAgreementPage, AddRedisDatabasePage } from '../../../pageObjects';
+import {
+    commonUrl,
+    ossStandaloneConfig
+} from '../../../helpers/conf';
+
+const myRedisDatabasePage = new MyRedisDatabasePage();
+const userAgreementPage = new UserAgreementPage();
+const addRedisDatabasePage = new AddRedisDatabasePage();
+const workbenchPage = new WorkbenchPage();
+
+const indexName = 'idx';
+const commandsForIndex = [
+    `FT.CREATE ${indexName} ON HASH PREFIX 1 product: SCHEMA price NUMERIC SORTABLE`,
+    'HMSET product:1 price 20',
+    'HMSET product:2 price 100'
+];
+
+fixture `Command results at Workbench`
+    .meta({type: 'regression'})
+    .page(commonUrl)
+    .beforeEach(async t => {
+        await t.maximizeWindow();
+        await userAgreementPage.acceptLicenseTerms();
+        await t.expect(addRedisDatabasePage.addDatabaseButton.exists).ok('The add redis database view', {timeout: 20000});
+        await addNewStandaloneDatabase(ossStandaloneConfig);
+        //Connect to DB
+        await myRedisDatabasePage.clickOnDBByName(ossStandaloneConfig.databaseName);
+        //Add index and data
+        await t.click(myRedisDatabasePage.workbenchButton);
+        for(let command of commandsForIndex) {
+            await workbenchPage.sendCommandInWorkbench(command);
+        }
```

Comment:
If all of this, of part of it, exists on other tests, let's make one function in common actions.

Reply:
done